pkgs/by-name/sa/satellite/package.nix

```
@@ -0,0 +1,57 @@
+{ lib
+, python3
+, fetchFromGitea
+, gobject-introspection
+, gtk3
+, libhandy
+, modemmanager
+, wrapGAppsHook
+}:
+
+python3.pkgs.buildPythonApplication rec {
```

```suggestion
python3Packages.buildPythonApplication rec {
```

every other usage should be changed to use this as well.

---

@SuperSandro2000 why not?

---

Because the two variables are identically here and I didn't want to send the author into another review round.

---

Is `python3Packages` preferred over `python3.pkgs`? If so, I'll use that instead next time.

---

There are discussions to remove python3.pkgs maybe eventually but this will be done by a treewide change. Continue to use what you prefer for now.